torch/csrc/jit/python/python_sugared_value.h

```diff
@@ -328,7 +328,10 @@ struct VISIBILITY_HIDDEN PythonClassValue : public ClassValue {
 struct VISIBILITY_HIDDEN PythonExceptionValue : public ExceptionValue {
   explicit PythonExceptionValue(const py::object& exception_class)
       : ExceptionValue(
-            py::str(py::getattr(exception_class, "__name__", py::str("")))) {}
+            py::str(py::getattr(exception_class, "__name__", py::str("")))) {
+    exception_class_qualified_name_ = py::str(py::module::import("torch._jit_internal")
+                                  .attr("_qualified_name")(exception_class));
```

Comment:
super nit:
This can be rewritten in to member initialization list like `ExceptionValue`.

Reply:
Actually maybe it is better if we use the un-mangled qualified name here. Since this is just going to be a string in the IR/output. There is no chance it is going to be confused with classes from other compilation units. What do you think?

Reply:
Sure, I'll add an optional argument to torch._jit_internal._qualified_name to control if mangling